code/modules/paperwork/pen.dm

```diff
@@ -147,19 +147,36 @@
 	var/on = FALSE
 	var/brightness_on = 2
 	light_color = LIGHT_COLOR_RED
-	stealthy_audio = TRUE
+	var/backstab_sound = 'sound/items/unsheath.ogg'
 	armour_penetration_flat = 20
 
 /obj/item/pen/edagger/attack(mob/living/M, mob/living/user, def_zone)
-	if(on && user.dir == M.dir && !HAS_TRAIT(user, TRAIT_FLOORED) && user != M)
-		M.apply_damage(12, BRUTE, def_zone)
-		M.apply_damage(40, STAMINA) //Just enough to slow
-		M.KnockDown(2 SECONDS)
-		M.visible_message("<span class='warning'>[user] stabs [M] in the back!</span>", "<span class='userdanger'>[user] stabs you in the back! The energy blade makes you collapse in pain!</span>")
-	. = ..()
-
-/obj/item/pen/edagger/get_clamped_volume()
-	return 5
+	if(on && HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
+		return
+	if(!on)
+		playsound(loc, 'sound/weapons/tap.ogg', get_clamped_volume(), 1, -1)
```

Comment:
Why did you remove the parent call and start copy and pasting it's code here? What was wrong with the way it was written before?

Reply:
Well, for the reactive armor, pretty sure you can just use the parent return value to decide whether or not to continue, since it gets set to `M.attacked_by`. With the way you have it now, the backstab damage is always applied no matter what, given the dagger is on. Also the parent `playsound` call already has `ignore_walls = TRUE`.